src/app/auth.service.ts

```diff
@@ -13,11 +13,13 @@ export class AuthService {
 
   public loggedInStatus: boolean;
 
+  private baseUri: string = `https://limitless-eyrie-83209.herokuapp.com/`;
```

Comment:
Just a suggestion, for later task.  Move end point urls to a new config.js, or constants.js files for better reusability.